spec/controllers/wishlists_for_guests_controller_spec.rb

```diff
@@ -0,0 +1,38 @@
+require "rails_helper"
+
+RSpec.describe WishlistsForGuestsController do
+  let(:new_wishlist) { Wishlist.create(name: "Name", description: "Description") }
+  let(:correct_token) { new_wishlist[:token] }
+  let(:incorrect_token) { "000"}
+
+  describe "GET show" do
+    context "when a valid token is provided" do
+      before do
+        get_request(correct_token)
+      end
+
+      it "has a 200 status code" do
+        expect(response.status).to eq(200)
+      end
+    end
+
+    context "when an invalid token is provided" do
+      before do
+        get_request(incorrect_token)
+      end
+
+      it "renders a 404 page" do
+        expect(response).to render_template(:file => "#{Rails.root}/public/404.html")
+      end
+
+      it "has a 200 status code" do
+        expect(response.status).to eq(200)
+      end
+    end
+  end
+
+  private
```

Comment:
I never add empty line after private or protected. But there are many different code styles, you should pick use for project and keep it global  (the same every time)